api/EXT_color_buffer_float.json

```diff
@@ -5,40 +5,40 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/EXT_color_buffer_float",
         "support": {
           "chrome": {
-            "version_added": "63"
+            "version_added": false
```

Comment:
Is the "EXT_color_buffer_float" extension not supported anywhere?

Seems like this is a problem with our tests. We create a WebGL1 context, and this extension is only supported for WebGL2.

I don't know if there are any WebGL1-only extensions, but if there aren't, then we should try to create a WebGL2 context and fall back to WebGL1. Or we create one of each and try both in these extension tests.

I won't review beyond this point because the results could all be wrong due to this issue.

Reply:
Ah, my bad, I thought this was a WebGL1 extension.  I'll get the collector fixed to also test WebGL2!

Reply:
I reverted the changes to this file and will defer them to a new PR when collector v0.5.3 is released.  I did a quick check of the other files, and have confirmed that the extensions missing in Edge are not exposed through WebGL2 (otherwise, all of the data is for browser versions before WebGL2 was introduced).